src/api/entities/Asset/Checkpoints/__tests__/Schedules.ts

```diff
@@ -1,8 +1,8 @@
 import BigNumber from 'bignumber.js';
-import { Ticker } from 'polymesh-types/types';
 import sinon, { SinonStub } from 'sinon';
 
 import { CheckpointSchedule, Context, Namespace, TransactionQueue } from '~/internal';
+import { Ticker } from '~/polkadot/polymesh';
```

Comment:
Why this change? Also, shouldn't this be `PolymeshPrimitivesTicker`?

Reply:
There's a ton of changes like this one. We should move to the long name types wherever possible IMO